kDriveCore/Data/UploadQueue/Servicies/PhotoLibraryUploader+Scan.swift

```diff
@@ -38,8 +38,7 @@ public extension PhotoLibraryUploader {
             options.sortDescriptors = [NSSortDescriptor(key: "creationDate", ascending: true)]
 
             let typesPredicates = getAssetPredicates(forSettings: settings)
-            let lastSyncDate = settings.lastSync as NSDate
-            let datePredicate = NSPredicate(format: "creationDate > %@ OR modificationDate > %@", lastSyncDate, lastSyncDate)
```

Comment:
Cuz right now, if a picture is aded or removed from the favorites, it will re-trigger an upload.
The expected behaviour is to re-trigger an upload only if a file is edited. This is a quickfix. The real patch will follow.